test/api.ts

```diff
@@ -119,7 +119,7 @@ suite('atomicassets', function () {
 
     test('get_assets', async function () {
         const res = await atomicassets.atomicassets.v1.get_assets({
-            collection_name: ['taco', 'alien.worlds'],
+            collection_name: [Name.from('taco'), 'alien.worlds'],
```

Comment:
Why are you wrapping one of the names and not the other one? 🤔

Reply:
I created the wrapping here to ensure `buildQueryParams` is working, leave 'alien worlds' as string to ensure string is acceptable.